types/members.ts

```diff
@@ -2,6 +2,7 @@ type MemberLevel = "Student" | "Individual Contributor" | "Leader";
 
 export interface MemberInterface {
   name: string; // full name, e.g. Frances Coronel
+  affiliation: string // company or school, eg. Google
```

Comment:
technically it also includes the title, right?

Reply:
sounds good, i was just thinking it might be worth updating the comment for that too

Reply:
also it should be optional since we are not requiring it

Reply:
made some updates!